usr/sbin/rear

```diff
@@ -363,6 +363,14 @@ hash -r
 # Make sure that we use only English:
 export LC_CTYPE=C LC_ALL=C LANG=C
 
+# Remember what TMPDIR was originally set when ReaR was launched
+# before TMPDIR may be set (if unset) to /var/tmp in default.conf
+# because testing if TMPDIR is '/var/tmp' would be insufficient
+# to determine if TMPDIR was set or unset when the user called 'rear'
+# because the user may have specified TMPDIR to be '/var/tmp'
+# cf. https://github.com/rear/rear/pull/3168#issuecomment-1997057144
+readonly TMPDIR_ORIG="${TMPDIR-}"
```

Comment:
Consider making `TMPDIR_ORIG` lowercase. Of course we are in the main program and not in a function, so we can't make the variable `local` - the variable is global. But the usage of the variable is local and not global - the intent is to use it only in this script and not in anything called by it. Cf. the variable `saved_tmpdir`.

Reply:
It is intentionally uppercase because
TMPDIR_ORIG is meant to be used (if needed)
also by any ReaR script.
For example by some third-party backup script
that may need to set some special TMPDIR.
E.g. TMPDIR_ORIG could be useful to find out
what the user's original intent about TMPDIR was,
e.g. to set a special TMPDIR in compliance
with the user's original intent of TMPDIR.